docs/source/support/contribution.rst

```diff
@@ -69,7 +69,7 @@ Alternately, you can install the development dependencies with pip:
 .. code-block:: bash
 
     $ pip3 install -e .[dev]
-    $ ./scripts/installation/install_solc.sh
+    $ ./scripts/installation/install_solc.py
```

Comment:
I assumed it's ok to run it like this since the script has a shebang line: `#!/usr/bin/env python3`
`pipenv run install-solc` seems like a much better choice.

Reply:
Ah, you are right about the shebang! - I missed that 👍

Reply:
Let's leave it as executing the python file then.